src/model/Stroke.cpp

```diff
@@ -450,13 +411,13 @@ void Stroke::setPressure(const vector<double>& pressure)
 	XOJ_CHECK_TYPE(Stroke);
 
 	// The last pressure is not used - as there is no line drawn from this point
-	if (this->pointCount - 1 > (int)pressure.size())
+	if (this->points.size() - 1 != pressure.size())
```

Comment:
Don't see any reason to change the condition. If we read a damaged file that has a mismatching pressure file, we should recover at least some of the pressure data instead of discarding all of it. (Yes, the previous condition refuses to load pressure information when the given data is larger than the number of points, but that could be improved by loading whatever can be loaded and then emitting a warning).

Reply:
mmh ok see your point. I'll look if I can make an even better decision. Or just revert that change